### PR TITLE
Update BAKTA: restructure to allow subcommands and add BAKTA/BAKTADBDOWNLOAD

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

Maybe that `-stub-run` is not being respected but I'm not sure why

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by jfy133** [comment]
@nf-core-bot prettier

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: modules/nf-core/bakta/baktadbdownload/main.nf [hunk 1]
```diff
@@ -0,0 +1,45 @@
+process BAKTA_BAKTADBDOWNLOAD {
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::bakta=1.6.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bakta:1.6.0--pyhdfd78af_0' :
+        'quay.io/biocontainers/bakta:1.6.0--pyhdfd78af_0' }"
+
```
Comment (nvnieuwk) [inline modules/nf-core/bakta/baktadbdownload/main.nf:8]: Interesting to see a module without input :)

**Review by nvnieuwk — APPROVED** [review]
LGTM!

File: modules/nf-core/bakta/baktadbdownload/main.nf (hunk 1 above)
Comment (jfy133) [reply modules/nf-core/bakta/baktadbdownload/main.nf:8]: It always looks... 'dirty' to me :laughing;

**Pull request merged**